docs/source/ADR.rst

```diff
@@ -1,4 +1,4 @@
-
+|RAT banner|
```

Comment:
![Screenshot 2023-02-16 101337](https://user-images.githubusercontent.com/34302892/219336876-e0baf74d-a4c5-444d-93d9-465cfa7e0187.png)
I think the header and footer are overkill and can become clutter. The logo in the sidebar and near the title are perfect

Reply:
I personally like the header but I agree using the RasCAL footer might be too cluttered for some pages. It looks okay in the user manual sections because they are long. However (like you pointed out), it looks very cluttered in Support and RAT Tutorial. I put the footer incase there was a need to connect RAT and RASCAL. However, I can remove the RasCAL footer from all the pages.